hack/multikueue-e2e-test.sh

```diff
@@ -80,16 +80,27 @@ function kind_load {
     install_jobset "$WORKER2_KIND_CLUSTER_NAME"
 
     # KUBEFLOW SETUP
+    # In order for MPI-operator and Training-operator to work on the same cluster it is required that:
+    # 1. 'kubeflow.org_mpijobs.yaml' is removed from base/crds/kustomization.yaml - https://github.com/kubeflow/training-operator/issues/1930
+    # 2. Training-operator deployment is modified to enable all kubeflow jobs except for mpi -  https://github.com/kubeflow/training-operator/issues/1777
+   
+    # Modify the `newTag` for the `kubeflow/training-operator` to use the one training-operator version
+    $YQ eval '(.images[] | select(.name == "kubeflow/training-operator").newTag) = env(KUBEFLOW_IMAGE_VERSION)' -i "$KUBEFLOW_MANIFEST_MANAGER/kustomization.yaml"
```

Comment:
Are you reverting these changes after testing? If not, users may unintentionally commit unnecessary changes to this file.

Like here:

https://github.com/kubernetes-sigs/kueue/blob/467b4ee707286f1f86f4f94cdb681111152bee61/hack/multikueue-e2e-test.sh#L109

https://github.com/kubernetes-sigs/kueue/blob/467b4ee707286f1f86f4f94cdb681111152bee61/hack/e2e-common.sh#L97-L99

Reply:
I don't, good point I will add the restore

Reply:
isn't this file outside of the tracked files?

Reply:
No. It's on the git.